django/db/backends/postgresql/operations.py

```diff
@@ -279,8 +279,8 @@ def window_frame_range_start_end(self, start=None, end=None):
         start_, end_ = super().window_frame_range_start_end(start, end)
         if (start and start < 0) or (end and end > 0):
             raise NotSupportedError(
-                'PostgreSQL only supports UNBOUNDED together with PRECEDING '
-                'and FOLLOWING.'
+                '%s only supports UNBOUNDED together with PRECEDING '
+                'and FOLLOWING.' % self.connection.display_name
```

Comment:
We can move this check to the `BaseDatabaseOperations.window_frame_range_start_end()` and use a new feature flag in it.